README.md

```diff
@@ -48,6 +48,51 @@ After that, you may run the binary as any normal program:
 ./shark-clipper-linux-x64-latest.bin
 ```
 
+#### Using the Mac Prebuilt Binary
+
+Mac requires additional security steps to run the prebuilt binary.
+These steps need to be done every time a new binary is downloaded and run for the first time.
+Running binary files is best done via the command line.
+
+This [command line primer](https://gomakethings.com/navigating-the-file-system-with-terminal/)
+includes instructions on running files and navigating directories.
+
+For a more comprehensive command line guide refer to MIT's [Missing Semester course](https://missing.csail.mit.edu/2020/course-shell/).
```

Comment:
This should be part of the previous paragraph.

Reply:
Sorry about multiple commits, I think I misunderstood this direction the first time.